src/ring_theory/ideal/operations.lean

```diff
@@ -342,17 +329,14 @@ variables {I J K L : ideal R}
 
 instance : has_mul (ideal R) := ⟨(•)⟩
 
-@[simp] lemma one_eq_top : (1 : ideal R) = ⊤ :=
-by erw [submodule.one_eq_range, linear_map.range_id]
+@[simp] lemma one_eq_top : (1 : ideal R) = ⊤ := by erw [submodule.one_eq_range, linear_map.range_id]
```

Comment:
This change I'm neutral-to-negative towards. The statement is mathematically uninteresting so I don't mind that we try to minimize the effort spent, but the proof is nontrivial enough to count as containing new ideas to me.

Perhaps a rule of thumb is that I only really want to see oneliners like this if they are corollaries (in the vague sense of the working mathematician).